typings/index.d.ts

```diff
@@ -1333,6 +1333,12 @@ declare module 'discord.js' {
 		public unban(user: UserResolvable, reason?: string): Promise<User>;
 	}
 
+	export class GuildBanStore extends DataStore<Snowflake, BanInfo> {
+		constructor(guild: Guild);
+		public add(ban: BanInfo, cache: boolean): BanInfo;
+		public fetch(options: FetchBanOptions): Promise<Collection<Snowflake, BanInfo> | BanInfo>;
```

Comment:
I think it would be better to add overloads for this method.
```ts
public fetch(): Promise<GuildBanStore>;
public fetch(options: { cache: false }): Promise<Collection<Snowflake, BanInfo>>;
public fetch(options: FetchBanOptions): Promise<BanInfo>;
```